api/server/router/image/backend.go

```diff
@@ -22,7 +22,7 @@ type Backend interface {
 type imageBackend interface {
 	ImageDelete(imageRef string, force, prune bool) ([]types.ImageDeleteResponseItem, error)
 	ImageHistory(imageName string) ([]*image.HistoryResponseItem, error)
-	Images(imageFilters filters.Args, all bool, withExtraAttrs bool) ([]*types.ImageSummary, error)
+	Images(ctx context.Context, opts types.ImageListOptions, withContainerCount bool) ([]*types.ImageSummary, error)
```

Comment:
LOL, not sure anymore on which PR I commented that; but given that we're adding a `ImageListOptions`, I think we should add the `withContainerCount` as field in that struct, and remove it as a separate boolean arg here